src/form/Select/Select.tsx

```diff
@@ -124,6 +124,14 @@ export interface SelectProps {
    */
   createable?: boolean;
 
+  /**
+   * Function to render the create option.
+   */
+  renderCreateOption?: (
```

Comment:
I'm wondering if we should put this on the `<SelectMenu />` component rather than the `Select` component itself. Probably not a big deal but just a thought to keep the primary API clean.

Reply:
We have it for both

Reply:
My recommendation is to make it where its like:

```
<Select 
   menu={
     <SelectMenu renderCreateOptoin={} />
    }
>
```

That makes the consumer api a little more messy though so not a big deal but helps keep the API isolated. Don't worry about it for now.

Reply:
Yes, I've got it, but then user need to override the menu for passing renderCreateOption. Probably that make sense